src/nxdoc/web-ui/web-ui-tutorials/web-ui-slots.md

```diff
@@ -2,24 +2,24 @@
 title: 'HOWTO: Customize Slots'
 description: Learn how to use Nuxeo Slots with Web UI.
 review:
-    comment: ''
-    date: '2017-12-15'
-    status: ok
+  comment: ''
+  date: '2017-12-15'
+  status: ok
 toc: true
 details:
-    howto:
-        excerpt: Learn how to use Nuxeo Slots with Web UI.
-        level: Advanced
-        tool: code
-        topics: UIElements
+  howto:
+    excerpt: Learn how to use Nuxeo Slots with Web UI.
+    level: Advanced
+    tool: code
+    topics: UIElements
 labels:
-    - lts2016-ok
-    - nuxeo-web-ui
-    - gbarata
-    - extension
-    - nuxeo-slot
-    - nuxeo-slot-content
-    - lts2017-ok
+  - lts2016-ok
+  - nuxeo-web-ui
+  - gbarata
+  - extension
+  - nuxeo-slot
+  - nuxeo-slot-content
+  - lts2017-ok
```

Comment:
Please be careful with spacing when updating the frontmatter, this could prevent the build from passing. @manonlumeau could confirm if this looks ok.